lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -258,74 +258,116 @@ def _get_converged_core_config(service_mconfig: object) -> bool:
 
 
 def _get_default_slice_service_type_config(service_mconfig: object) -> str:
-    """
-    Retrieves default_slice_service_type config value,If it does not exist
-    it defaults to 1.
+    """Retrieve default_slice_service_type config value. If it does not exist, it defaults to 1.
+
     Args:
-        service_mconfig:
+        service_mconfig: This is a configuration placeholder for mme.
 
-    Returns: slice service type value
+    Returns:
+        slice service type value.
     """
+    enable_default_service_slice_type = get_service_config_value(
+        'mme', 'default_slice_service_type', None,
+    )
+
+    if enable_default_service_slice_type is not None:
+        return enable_default_service_slice_type
+
     return service_mconfig.default_slice_service_type or DEFAULT_NGAP_S_NSSAI_SST
 
 
 def _get_default_slice_differentiator_type_config(service_mconfig: object) -> str:
-    """
-    Retrieves default_slice_differentiator config value,If it does not exist
-    it defaults to 0xffffff.
+    """Retrieve default_slice_differentiator config value. If it does not exist it defaults to 0xffffff.
```

Comment:
same here, remove the value and use the variable name

Reply:
Comment addressed.